pkg/swarm/distance_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package swarm
+
+import (
+	"testing"
+)
+
+type distanceTest struct {
+	x      []byte
+	y      []byte
+	result string
+}
+
+type distanceCmpTest struct {
+	a      []byte
+	x      []byte
+	y      []byte
+	result int
+}
+
+var (
+	distanceTests = []distanceTest{
+		{
+			x:      MustParseHexAddress("9100000000000000000000000000000000000000000000000000000000000000").Bytes(),
+			y:      MustParseHexAddress("8200000000000000000000000000000000000000000000000000000000000000").Bytes(),
+			result: "8593944123082061379093159043613555660984881674403010612303492563087302590464",
```

Comment:
Oh, an _easily_ verifiable number. Any reason why we have to run these testes with 32 byte ints?

Reply:
you mean why we _don't_?